index.md

```diff
@@ -3,7 +3,7 @@
 
 # Introduction
 
-The global climate crisis impacts all of us. From this, we know we should reduce our carbon emissions. However, if we do computing as part of our day-to-day work, it can form a significant part of our environmental impact, both for us personally and for our organisation(s). In particular, if you or your group does intensive computing, making use of high-performance computers or cloud resources, for example to train intricate models or run complex simulations, the carbon footprint can be sizable.
+As reports on X, X, and X have shown, he global climate crisis impacts all of us. From this, reducing carbon emissions has emerged as a means of addressing the climate crisis. For those of us that do computing as part of our day-to-day work, it can form a significant part of our environmental impact, both for us personally and for our organisation(s). In particular, if you or your group does intensive computing, making use of high-performance computers or cloud resources, for example to train intricate models or run complex simulations, the carbon footprint can be sizable.
```

Comment:
```suggestion
As reports on X, X, and X have shown, the global climate crisis impacts all of us. From this, reducing carbon emissions has emerged as a means of addressing the climate crisis. For those of us that do computing as part of our day-to-day work, it can form a significant part of our environmental impact, both for us personally and for our organisation(s). In particular, if you or your group does intensive computing, making use of high-performance computers or cloud resources, for example to train intricate models or run complex simulations, the carbon footprint can be sizable.
```

Reply:
Also, I'll look into good reports to reference here when I get a spare moment, hopefully by the end of the week. Unless you have ideas already, in which case feel free to do a commit or PR to add those!